src/include/sof/lib_manager.h

```diff
@@ -126,10 +126,6 @@ struct ext_library {
 	uint32_t lib_notif_count;
 
 	void *runtime_data;
-#if CONFIG_LIBRARY_AUTH_SUPPORT
```

Comment:
Just to confirm my understanding - this context is currently created and destroyed for each module load event but stored globally ? This change just locally allocates and assigns the context on each load ?